react/features/dynamic-branding/functions.web.ts

```diff
@@ -45,28 +46,23 @@ export function createMuiBrandingTheme(customTheme: any) {
         overwriteRecurrsive(newBreakpoints, customBreakpoints);
     }
 
-    let newSpacing = [ ...spacing ];
+    let newSpacing: any = [ ...spacing ];
 
     if (customSpacing && customSpacing.length) {
         newSpacing = customSpacing;
     }
 
-    return createMuiTheme({
-        props: {
-            // disable ripple effect on buttons globally
-            MuiButtonBase: {
-                disableRipple: true
-            }
-        },
-
-        // use token spacing array
-        spacing: newSpacing
-    }, {
+    return createTheme(adaptV4Theme({
```

Comment:
This is the recommended way to migrate from V4 to V5. I agree that we should try to get rid of it, but I'd rather do that in a future PR, as it may break unexpected things